UKHomeOffice/cerberus-service
# COP-11439: showing assign to me filter for task that are in progress tab

## Description
Show Assign to me filter(Checkbox) for tasks that are in progress tab

## To Test
1. Pull and run against dev
2. Click In progress tab on task list page to see Assign to me filter
3. Check Assign to me checkbox and click on apply button. Only the tasks assigned to you would be visible

![image](https://user-images.githubusercontent.com/1534608/184857151-966a04f8-2ce2-4bd9-b4a3-fd5fe1383194.png)

![image](https://user-images.githubusercontent.com/1534608/184857185-c14aa41c-b9fb-484b-bbe2-04fc93b720f9.png)


## Developer Checklist

\* Required

- [*] Up-to-date with main branch? \*

- [*] Does it have tests?

- [*] Pull request URL linked to JIRA ticket? \*

- [*] All reviewer comments resolved/answered? \*
